crates/forge/src/warn.rs

```diff
@@ -63,3 +60,29 @@ pub(crate) async fn warn_if_incompatible_rpc_version(
 
     Ok(())
 }
+
+fn snforge_std_version_requirement() -> VersionReq {
```

Comment:
Where was it before?

Reply:
`crates/forge/src/main.rs`

it's moved to `warn.rs`